my_main.py

```diff
@@ -0,0 +1,67 @@
+# HMMMM what should i do
+# I think book-reading should be an easy one. Or a math computation of some kind.
```

Comment:
Suggestion: Remove unnecessary comments

Reply:
And nice work on suggestion/ required/question modes for your comments